src/player.h

```diff
@@ -59,7 +59,7 @@ namespace Player {
 	/**
 	 * Initializes EasyRPG Player.
```

Comment:
Nit: there's a good chance for adding updated Doxygen lines here and for Game_Config